### DISCUSSION TITLE
杜比视界黑屏

### DISCUSSION THREAD

**Discussion by Czerror** [hooke007/mpv_PlayKit#277] [mpv 通用]
播放双层杜比视界时,不删除watch_later,第二次播放会黑屏报错,这是什么原因啊

[mpv-lazy.log](https://github.com/hooke007/MPV_lazy/files/10763522/mpv-lazy.log)
这是日志

**Comment by hooke007** [comment]
我无源验证，另外gpu-next相关的东西先升级播放器到最新版本再测一遍。

**Reply by Czerror** [reply]
已经更新到最新版本了,
![QQ截图20230217134359](https://user-images.githubusercontent.com/58031342/219559470-4cb66f2a-2df2-412b-b014-76ca83822b36.png)
![QQ截图20230217134410](https://user-images.githubusercontent.com/58031342/219559474-d4756619-72e6-456d-99d2-27f68caeafd2.png)
看起来像是解码错误,但是 删除播放记录又可以放

**Reply by Czerror** [reply]
magnet:?xt=urn:btih:514372BDA8E6A648ED587B6B582E2F4B65B5A444

最后生还者的双杜比

**Reply by hooke007** [reply]
能复现了，只要从中途播出启动就会崩解码

**Reply by Czerror** [reply]
对的  这个该怎么解决啊?

**Reply by hooke007** [reply]
不是我能解决的问题 https://github.com/mpv-player/mpv/issues/11322

**Reply by Czerror** [reply]
QAQ  等更新了

**Reply by dyphire** [reply]
~~这似乎是个很早就存在的恶性bug了，追溯到去年年中的mpv版本也一样可以复现，再往前的版本暂未测试~~

**Reply by dyphire** [reply]
又拿其他组的最后生还者的DOVI版web-dl测试了下，无法复现此问题。我怀疑有问题的是DVSUX这个组的web-dl，文件的时间码上很可能存在某些奇怪的错误

顺便这不是双层杜比视界，只是profile5版的BL+RPU的单层杜比视界，双层杜比视界是profile7的BL+RPU+EL

**Reply by Czerror** [reply]
> 又拿其他组的最后生还者的DOVI版web-dl测试了下，无法复现此问题。我怀疑有问题的是DVSUX这个组的web-dl，文件的时间码上很可能存在某些奇怪的错误
> 
> 顺便这不是双层杜比视界，只是profile5版的BL+RPU的单层杜比视界，双层杜比视界是profile7的BL+RPU+EL

那就是片源的问题?  我用电视播又没问题
